package.json

```diff
@@ -35,9 +35,34 @@
     "chalk": "^2.4.2",
     "shelljs": "^0.8.3"
   },
-  "devDependencies": {
-    "@types/chai": "^4.2.7",
+  "devDependencies": {        
+    "@types/archiver": "^3.0.0",
+    "@types/chai": "^4.2.9",
+    "@types/cli-table":"^0.3.0",    
+    "@types/inquirer": "^6.5.0",
+    "@types/fast-levenshtein": "^0.0.1",
+    "@types/fs-readdir-recursive": "^1.0.0",
+    "@types/fs-extra": "^8.1.0",
+    "@types/get-folder-size":"^2.0.0",
+    "@types/got":"^9.6.9",
+    "@types/glob":"^7.1.1",
+    "@types/lodash": "^4.14.149",
     "@types/mocha": "^5.2.7",
+    "@types/mustache": "^0.8.32", 
+    "@types/ncp": "^2.0.3",   
```

Comment:
Some added trailing whitespace on _several_ new lines in this file. Could you remove it before merge please?
Also: Please ensure _one_ space character after `:` (will be autoformatted by Yarn/npm otherwise).